.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,33 @@
+
+#### Description
+
+**Summary of the change**: Provide a concise summary of what this PR is about. Describe the changes made in a single sentence or short paragraph.
+
+**Detailed description**:
+- **What**: Detail what changes have been made in the PR.
+- **Why**: Explain the reasons behind the changes. Why were they necessary?
+- **How**: Describe how the changes were implemented, including any key aspects of the code modified or new features added.
+
+---
+
+#### Type of Change
+
```

Comment:
I was thinking about suggesting a checklist for affected databases in this section, or as an item under "Detailed description".
But I don't think it's absolutely necessary (compared to its significant value in a YouTrack issue template for example). Most likely users will include any specific database, if applicable, in the context of the description above.